src/ru/skypro/Main.java

```diff
@@ -4,5 +4,74 @@
 
     public static void main(String[] args) {
 	// write your code here
+        // Задание №1
+        int clientOS = 0;
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        }
+        if (clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+
+        // Задание №2
+        int OS = 0;
+        int clientDeviceYear = 2015;
+        if (OS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+            if (clientDeviceYear < 2015) {
+                System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+            } else if (OS == 1) {
+                System.out.println("Установите версию приложения для Android по ссылке");
+                if (clientDeviceYear >= 2015) {
+                    System.out.println("Установите облегченную версию приложения для Android по ссылке");}
+            }
+        }
+        // Задание №3
+        int year = 2022;
+        if ((year % 4 == 0) && ((year % 100 !=0) || year % 400 == 0)) {
+            System.out.println(year + " год является високосным");
+        } else {
+            System.out.println(year + " год не является високосным");
+        }
+
+        // Задание №4
+        int deliveryDistance = 95;
+        if (deliveryDistance <= 20) {
+            System.out.println("Потребуется дней: 1");
+        } else if (deliveryDistance > 20 && deliveryDistance <= 60) {
+            System.out.println("Потребуется дней: 2");
+        } else if (deliveryDistance > 60 && deliveryDistance <= 100) {
+            System.out.println("Потребуется дней: 3");
```

Comment:
можно упростить, убрав левые условие, поскольку в рамках одного условного оператора они будут автоматически выполняться из-за невыполнения вышележащих условий
```suggestion
        if (deliveryDistance <= 20) {
            System.out.println("Потребуется дней: 1");
        } else if (deliveryDistance <= 60) {
            System.out.println("Потребуется дней: 2");
        } else if (deliveryDistance <= 100) {
            System.out.println("Потребуется дней: 3");
```